tianshou/policy/modelfree/discrete_sac.py

```diff
@@ -109,7 +109,7 @@ def forward(  # type: ignore
         logits, hidden = self.actor(batch.obs, state=state, info=batch.info)
         dist = Categorical(logits=logits)
         if self.deterministic_eval and not self.training:
```

Comment:
pls adjust the docstring of what happens when `deterministic_eval=True` is passed

Reply:
done, check please